### DISCUSSION TITLE
Changing the duration of  caret blinking

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#1512] [Ideas] [answered]
It would be nice to change the duration of the caret blink. Say making the blinking slower or turning it off entirely.

**Comment by frogfrogfrogfrogfrog** [comment, marked as answer]
I don't think turning off the blinking would help. From what I know the reason it blinks is so that you can see the letter below the caret and making the blinking slower is sort of niche as I don't see what that would accomplish.
